docs/pages/includes/metrics.mdx

```diff
@@ -67,21 +67,34 @@
-| `grpc_client_started_total`               | counter | Teleport Proxy | Total number of RPCs started on the client.                                                                                     |
-| `grpc_client_handled_total`               | counter | Teleport Proxy | Total number of RPCs completed on the client, regardless of success or failure.                                                 |
-| `grpc_client_msg_received_total`          | counter | Teleport Proxy | Total number of RPC stream messages received on the client.                                                                     |
-| `grpc_client_msg_sent_total`              | counter | Teleport Proxy | Total number of gRPC stream messages sent by the client.                                                                        |
-| `proxy_connection_limit_exceeded_total`   | counter | Teleport Proxy | Number of connections that exceeded the Proxy Service connection limit.                                                         |
-| `proxy_ssh_sessions_total`                | gauge   | Teleport Proxy | Number of active sessions through this Proxy Service instance.                                                                  |
-| `proxy_missing_ssh_tunnels`               | gauge   | Teleport Proxy | Number of missing SSH tunnels. Used to debug if Nodes have discovered all Proxy Service instances.                              |
-| `remote_clusters`                         | gauge   | Teleport Proxy | Number of inbound connections from leaf clusters.                                                                               |
-| `teleport_connect_to_node_attempts_total` | counter | Teleport Proxy | Number of SSH connection attempts to a node. Use with `failed_connect_to_node_attempts_total` to get the failure rate.          |
-| `teleport_reverse_tunnels_connected`      | gauge   | Teleport Proxy | Number of reverse SSH tunnels connected to the Teleport Proxy Service by Teleport instances.                                    |
-| `trusted_clusters`                        | gauge   | Teleport Proxy | Number of outbound connections to leaf clusters.                                                                                |
+| Name                                      | Type      | Component      | Description                                                                                                                     |
+| ----------------------------------------- | --------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| `failed_connect_to_node_attempts_total`   | counter   | Teleport Proxy | Number of failed SSH connection attempts to a Node. Use with `teleport_connect_to_node_attempts_total` to get the failure rate. |
```

Comment:
Does `Node` refer to an SSH Service instance or any agent? We're phasing out the "Node" terminology, and the "Core Concepts" page doesn't mention this term.